draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -796,7 +794,7 @@ Further details of the algorithm are given in the following sections.
 
 ## Message identifiers
 
```

Comment:
As noted the ability to extract primary and secondary identifiers from a packet does not necessarily imply that the entire DNS payload was well formed